### ISSUE TITLE
v 5.7.2 on VSCode, command 'fsi.SendSelection' not found

### ISSUE THREAD

**Issue by Bircck** [ionide/ionide-vscode-fsharp#1578]
### command 'fsi.SendSelection' not found
TLDR : bunch of students at DTU has "command 'fsi.SendSelection' not found" error on ionide version 5.7.2

I'm part of a study course Functional programming on DTU denmark and there seems to be a bunch of student including myself having problems with F# during lab classes. I'm assuming this is because of the latest verison of F# as everything else including dotnet and visual studio code shows no errors when checked through command prompt. It's not possible to run F# when using the ionide extension

![image](https://user-images.githubusercontent.com/55695195/132831640-25164728-b8d8-400d-985c-20489f4bc40b.png)

## Machine info
 - OS: Windows
 - .NET SDK version: [e.g. 5.0.400]
 - Ionide version: [e.g 5.2.7 ]

**Comment by ricardozimmer** [comment]
Going back to 5.7.1 works fine.

**Comment by Krzysztof-Cieslak** [comment]
Hey,

We are really sorry for the issue. As mentioned above reversing to `5.7.1` is only currently known workaround. 

Also, as a side note, it's really cool to know that some folks are using Ionide at DTU - it's the place where I've started my F# adventure 😀

**Comment by Krzysztof-Cieslak** [comment]
We will track this in #1576
